bridge/cypress/integration/uniform-integrations.spec.ts

```diff
@@ -723,4 +723,29 @@ describe('Edit subscriptions', () => {
       .shouldHaveServices([service])
       .assertIsUpdateButtonEnabled(true);
   });
+
+  it('should show danger zone on edit page', () => {
+    interceptSubscription(integrationID, subscriptionID, 'sockshop', 'carts', 'dev');
+
+    uniformPage.visitEdit(integrationID, subscriptionID);
+
+    cy.get('ktb-danger-zone').should('exist');
+  });
+
+  it('should show deletion dialog after clicking the deleteSubscription button', () => {
+    interceptSubscription(integrationID, subscriptionID, 'sockshop', 'carts', 'dev');
+
+    uniformPage.visitEdit(integrationID, subscriptionID).openDeletionDialog();
+
+    cy.get('ktb-deletion-dialog').should('exist');
```

Comment:
Could you either move the deletion-dialog check to the related page object or to the global commands?